study.md

```diff
@@ -151,7 +151,11 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+1. GET: Go get and display data (usually a webpage).
+2. PUT: Used to request changes or make new things on a particular webpage.
+3. DELETE: Opposite of PUT, when you want to request things to be deleted.
+4. POST: Similar to PUT but for when you want an action to be repeated. Still a bit blurry on this one.
```

Comment:
So POST is actually how we will create a new resource, while PUT and PATCH are used to modify or update an existing resource.